src/main.rs

```diff
@@ -160,7 +160,7 @@ fn main() {
                 .help(
                     "Network parameters \"mac=<mac_addr>,\
                      sock=<socket_path>, num_queue_pairs=<number_of_queue_pairs>,\
-                     queue_size=<size_of_each_queue>\"",
+                     queue_size=<size_of_each_queue>, num_vectors=<number_of_vectors>\"",
```

Comment:
Why do we expect the user to provide the number of vectors since as you explain it in your commit message, you know how to compute it from the number of queues.

Reply:
Yes, it seems this option could be removed from the command line and config.rs, only need to compute it in vm.rs.